app/Services/GenreService.php

```diff
@@ -15,4 +15,15 @@ public function findAll()
     {
         return Genre::all();
     }
+
+    /**
+     * Find movie from name.
+     *
+     * @param string $title
+     * @return \Illuminate\Http\Response
+     */
+    public function getByTitle(string $name)
+    {
+        return Genre::whereRaw("name LIKE '" . $name . "'")->firstOrFail();
```

Comment:
Mozes i ovako: 
```suggestion
        return Genre::where('name', 'LIKE' '" . $name . "'")->firstOrFail();
```
Posto onako moze neko sql injection ako ne escapujes :D